app/src/main/java/com/bucketlist/MainActivity.kt

```diff
@@ -13,15 +13,15 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.main_activity)
         if (savedInstanceState == null) {
             supportFragmentManager.beginTransaction()
-                .replace(R.id.container, MainFragment.newInstance())
+                .replace(R.id.container, MainFragment.newInstance(),)
```

Comment:
Adding trailing comma as recommended here: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

Reply:
Hmmm, this is interesting.  I appreciate the best practice link, as I didn't realize this was an acceptable convention.